plugins/feedback/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@janus-idp/backstage-plugin-feedback",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
will this be private?

Reply:
No it does not need to be private, I'll remove it thanks for pointing it out.